site/docs/v1/tech/guides/advanced-registration-forms.adoc

```diff
@@ -462,6 +479,138 @@ image::guides/advanced-registration-forms/flask-app-screen-shot.png[The user pro
 
 You can view the link:https://github.com/FusionAuth/fusionauth-example-flask-portal[example code here], which includes templates and the login and registration links as well as the above profile modification code.
 
+== Editing User Data In the Admin UI
+
+[NOTE.since]
+====
+Available Since Version 1.20.0
+====
+
+Advanced user registration forms add custom data to your users' profiles. However, what happens when that profile data needs to be modified? You can write code against the APIs to modify it, but using a custom admin form is easier. You don't have to write any code, only configure a form or two.
+
+There are two types of profile data:
+
+* User data, which is associated with the user. This could be in standard fields such as `mobilePhone` or custom data fields in `user.data`.
+* Registration data, associated with the user's registration to an application. This could be in standard fields such as `roles` or custom data in `registration.data`.
+
+Each of these types of profile data has an admin form associated with it. Admin user forms are associated with the tenant and admin registration forms are associated with an application. 
+
+The default user and registration editing forms ship with FusionAuth and are implemented using this functionality. They can easily be replaced by your own custom forms suited to your business needs.
+
+=== How To Use A Custom Admin Form
+ 
+There are a few steps to using custom admin forms:
+
+* Determine if you are going to create a custom registration form, user form or both.
+* Consider data sources for each field of the profile. It could be user registration, API calls from other systems, or manually entered by an admin. Should some data be protected from admin modification?
+* Create custom fields if needed. This is where you'd set the data type, form control and validation rules.
+* Assemble the fields into a form, including organizing them into steps.
```

Comment:
Should we use sections instead of steps here?

> ... including organizing them into sections.

Reply:
Great catch, thanks!